AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -112,6 +112,12 @@ techadvisor.co.uk,forum.notebookreview.com,computerworld.com,skysports.com#$#bod
 !
 !
 !
+manutd.com#@#.cookie-setting-message
+manutd.com#@#.cookie-policy-message
+manutd.com#$#.cookie-setting-message { display: none !important; }
+manutd.com#$#.cookie-policy-message { display: none !important; }
+manutd.com#$##tint-overlay { display: none !important; }
+manutd.com#$#body { overflow: auto !important; }
```

Comment:
Please move it to the whitelist file with task link.

```
manutd.com#@#.cookie-setting-message
manutd.com#@#.cookie-policy-message
```

I guess this will be ok instead of the css rules.

```
manutd.com#%#//scriptlet('set-cookie', 'userConsentCookiePolicy', 'true')
```

Reply:
Applied.
https://github.com/AdguardTeam/AdguardFilters/pull/104221/commits/cfc63751d64f10728567e0313442e8dd1a94a30f
https://github.com/AdguardTeam/AdguardFilters/pull/104221/commits/0f8601502cc0b69e12f655453e15e4dae08c4b5a
https://github.com/AdguardTeam/AdguardFilters/pull/104221/commits/916d8ef48da0e9b38566be60bd9b0f6f89c99774